kustomize/kustomization.yaml

```diff
@@ -21,8 +21,14 @@ components:
 # - components/google-cloud-operations
 # - components/memorystore
 # - components/network-policies
+# - components/non-public-frontend
 # - components/service-accounts
+# - components/single-shared-session
 # - components/spanner
+# - components/service-mesh-istio
+# - components/without-loadgenerator
```

Comment:
I removed the duplicate without-loadgenerator and confirmed they work in the listed order. native-grpc-health-check was not duplicated and I think it needs to be at the end since it also modifies the image tags. But I could be wrong there.